pom.xml

```diff
@@ -86,6 +86,7 @@
         <exec-maven-plugin.version>3.0.0</exec-maven-plugin.version>
         <owasp-dependency-check-maven.version>7.1.2</owasp-dependency-check-maven.version>
         <lombok.version>1.18.20</lombok.version>
+        <casdoor.version>1.6.0</casdoor.version>
```

Comment:
Should add this to `dolphinscheduler-bom` module.

Reply:
Done